transmission-ls.json

```
@@ -0,0 +1,63 @@
+{
+    "Transmission LS": {
+        "description": "Open source bittorrent client powered by linuxserver.io",
```

```suggestion
"description": "Open source bittorrent client powered by linuxserver.io<p>Based on a custom docker image: <a href='https://hub.docker.com/r/linuxserver/transmission' target='_blank'>https://hub.docker.com/r/linuxserver/transmission</a>, available for amd64 and arm64 architecture.</p>
```
Adding in the supported architectures and whether it's an official or custom docker image being used